src/UI/templates/js/Core/src/core.js

```diff
@@ -2,7 +2,7 @@ import replaceContent from './core.replaceContent.js'
 import Tooltip from './core.Tooltip.js'
 
 if (typeof il === 'undefined') {
-    il = {}
```

Comment:
@thibsy / @lzehnder please revert those changes since these are not part of this refactoring